.pipelines/azure_pipeline_prod.yaml

```diff
@@ -88,12 +97,25 @@ steps:
       az account set -s ${{ variables.subscription }}
       az acr login -n ${{ variables.containerRegistry }}
 
-      docker buildx build --platform linux/amd64,linux/arm64 --tag ${{ variables.repoImageName }}:ciprod-$(datetime)-$(commit) -f kubernetes/linux/Dockerfile.multiarch --metadata-file $(Build.ArtifactStagingDirectory)/metadata.json --build-arg IMAGE_TAG=ciprod-$(datetime)-$(commit) --push .
+      if [ "$(Build.Reason)" != "PullRequest" ]; then
+        docker buildx build --platform linux/amd64,linux/arm64 --tag ${{ variables.repoImageName }}:ciprod-$(datetime)-$(commit) -f kubernetes/linux/Dockerfile.multiarch --metadata-file $(Build.ArtifactStagingDirectory)/metadata.json --build-arg IMAGE_TAG=ciprod-$(datetime)-$(commit) --push .
+
+        docker pull ${{ variables.repoImageName }}:ciprod-$(datetime)-$(commit)
+      else
+        docker buildx build --platform linux/amd64,linux/arm64 --tag ${{ variables.repoImageName }}:ciprod-$(datetime)-$(commit) -f kubernetes/linux/Dockerfile.multiarch --metadata-file $(Build.ArtifactStagingDirectory)/metadata.json --build-arg IMAGE_TAG=ciprod-$(datetime)-$(commit) .
```

Comment:
I see the format of the image tag is, ciprod-<datetime>-commit in non-Pull request scenario, but the image tag we would need whatever release tag passed to as part of the release, right?

Reply:
Yes this tag is for pushing to our middle man ACR. For release we will use the release tag which is taken as a variable input when we trigger the release